lib/aytests/image_builder.rb

```diff
@@ -18,9 +18,9 @@ class ImageBuilder
     IMAGE_BOX_NAME = "autoyast_vagrant_box_image_0.img".freeze
     SLEEP_TIME_AFTER_UPGRADE = 150
     SLEEP_TIME_AFTER_SHUTDOWN = 15
-    SSH_USER = "root".freeze
-    SSH_PASSWORD = "nots3cr3t".freeze
-    SSH_PORT = "22".freeze
+    SSH_USER = "root"
```

Comment:
I see, the problem is here: https://github.com/busyloop/net-ssh-simple/blob/3bc03322d2cd69d97782c3c79045191f2d2428b1/lib/net/ssh/simple.rb#L635